erroranalysis/tests/test_cohort_filter.py

```diff
@@ -83,6 +83,27 @@ def test_cohort_filter_true_y(self):
                            model_task,
                            filters=filters)
 
+    def test_cohort_filter_string_encoded_true_y(self):
```

Comment:
This test looks very identical to `test_cohort_filter_true_y`. Could you reuse the test `test_cohort_filter_true_y` and parametrize it?

Reply:
done, parameterized it